lib/connectors/memory.js

```diff
@@ -312,8 +351,7 @@ function getValue(obj, path) {
   return val;
 }
 
-Memory.prototype.all = function all(model, filter, options, callback) {
-  var self = this;
+Memory.prototype._findAll = function(model, filter) {
```

Comment:
IIRC, this new method ignores `filter.include`, which is not clear from the name. Please rename, e.g. to something like `_findAllSkippingIncludes`